lib/smart_answer_flows/marriage-abroad/outcomes/_appointment_documents_in_vietnam.govspeak.erb

```diff
@@ -1,9 +1,20 @@
-- a [service request form](/government/publications/service-request-form)
-- a [notice of marriage form](/government/publications/notice-of-same-sex-marriage-vietnam)
-- an [affirmation](/government/publications/affirmation-of-marital-status) or [affidavit](/government/publications/affidavit-of-marital-status) of marital status
+<% if calculator.ceremony_country == 'vietnam' &&
```

Comment:
if the partial is named: `_appointment_documents_in_vietnam` why do we need this check?